css/src/components/checkbox.scss

```diff
@@ -0,0 +1,121 @@
+/* stylelint-disable selector-max-specificity, max-nesting-depth, selector-no-qualifying-type */
+
+$checkbox-border-radius: $radius-sm !default;
+$checkbox-border-color: $text !default;
+$checkbox-check-hover-color: $secondary !default;
```

Comment:
audit for unused variables?

Reply:
Good catch, `$checkbox-check-hover-color` is now used in the checkmark mixin.